src/index.ts

```diff
@@ -18,19 +18,24 @@ interface GetOneHooks {
   after: (data: any) => Promise<any> | any;
 }
 
-interface WriteHooks {
+interface CreateHooks {
   before?: (data: any) => Promise<any> | any;
   after?: (data: any) => Promise<any> | any;
 }
 
+interface UpdateHooks {
+  before?: (data: any, record: any) => Promise<any> | any;
```

Comment:
Is it possible to use sequelize types?

Reply:
You're right. I'll adjust the code accordingly.